### ISSUE TITLE
Malformed directory name for swagger-schema-official when running flow-typed install

### ISSUE THREAD

**Issue by ilyabo** [flowtype/flow-typed#913]
• Found flow-bin@v0.42.0 installed. Installing libdefs compatible with this version of Flow...
• Found 87 dependencies in package.json. Searching for libdefs...
• rebasing flow-typed cache...done.
UNCAUGHT ERROR: Error: swagger-schema-official_v2.x.x-: 'swagger-schema-official_v2.x.x-' is a malformed definitions/npm/ directory name! Expected the name to be formatted as <PKGNAME>_v<MAJOR>.<MINOR>.<PATCH>

**Comment by rhiokim** [comment]
In my case, this issue was solved after install flow-typed@2.1.2

**Comment by glandre** [comment]
Any update on this issue?

**Comment by marudor** [comment]
It could help if you delete your local flow-typed folder and retry.

**Comment by derekclair** [comment]
I was able to get around this blocking error with v2.1.2, which according to @StefanoMagrassi is [the current stable version](https://github.com/flowtype/flow-typed/issues/905#issuecomment-305404297). `npm install -g flow-typed@2.1.2` if anyone else is having the same issue.

**Comment by mull** [comment]
Ran into the same error. Updating flow-typed to latest (2.1.2) fixed my issues.

**Comment by glandre** [comment]
v2.1.2 solved my problem.

**Comment by bsonntag** [comment]
Updating to 2.1.2 is not a solution for me, since [flow-typed hangs in Node 8](https://github.com/flowtype/flow-typed/issues/909).

**Comment by renatorib** [comment]
```
> flow-typed version
flow-typed@2.1.2
```
```
> flow-typed install
• Found 56 dependencies in package.json to install libdefs for. Searching...
• rebasing flow-typed cache...
UNCAUGHT ERROR: Error: flow_v0.38.x <=v0.46.x: Unexpected trailing characters:  <=v0.46.x

**Comment by marudor** [comment]
I've just released 2.1.3. Node8 should no longer hang
